### ISSUE TITLE
UX issue when creating event.

### ISSUE THREAD

**Issue by NobleO11** [werner-scholtz/kalender#79]
Hello, there is a small issue with the MonthView.
user now has to tap on a specific small area compared to the whole day grid.

please check following screenshot : 

![Screenshot (33)](https://github.com/werner-scholtz/kalender/assets/78362115/e29515cd-5bac-46c5-a158-e69042f979a7)

in empty days that has no events i will have to tap on a small area as shown in the screenshot above in red,

I think this is a big UX issue, I think if you could make the whole day clickable it would be perfect

**Comment by werner-scholtz** [comment]
Hi this will be fixed in the next release, it is a major overhaul of the package, I have not had any spare time to work on the package recently but I will get to it.

**Comment by NobleO11** [comment]
thank you so much.

**Comment by werner-scholtz** [comment]
Just a rough idea for how to fix this:
The plan is to get rid of the scroll-able view inside each week and to replace it with something similar to what google calendar does.
So once the events do not fit into the vertically space a button is placed at the bottom of the space, once clicked a overlay is displayed with the hidden events.

**Comment by NobleO11** [comment]
yes this is good, a button to be shown when a day is empty will do the job

**Comment by werner-scholtz** [comment]
Hi started work on this, not done yet but getting there see https://github.com/werner-scholtz/kalender/issues/133#issuecomment-2768481931
